README.md

```diff
@@ -1,2 +1,21 @@
-# Martian-Harmony
-Srishti 2019
+# Martian-Harmony-4.0
+The idea was to build a robot that can play musical instruments in the same fashion as humans. It consists of various instruments like Flute, Keyboard, Guitar, etc. All these instruments are being played using different mechanisms.
+## Team Members:

```

Comment:
Write the name in Alphabetical Order for team members as well as mentors

Reply:
@visvash  Following changes are needed 
1. Make a folder name **src** inside flute as well as piano folder and put all the source code files inside it.
2.  Make a folder of name **Mechanical Design** inside flute as well as piano put a llt eh solid works file inside that folder.
3. Make a common folder name **Images and videos** and pull all the videos and images inside that folder with proper title.
4. See the rules for making the readme file. Many points are missing in the readme file and subsections are not in proper order.

Note: **Refer the guidelines properly**